library/src/test/java/com/chuckerteam/chucker/api/ChuckerInterceptorTest.kt

```diff
@@ -51,6 +51,10 @@ class ChuckerInterceptorTest {
             override fun create(): File {
                 return File(tempDir, "testFile")
             }
+
+            override fun createExportFile(): File {
```

Comment:
Nit: it is not a big issue, since Chucker is a debug tool and the file is small, but in the scenario you presented we are creating the file and not removing it after sharing. We can leave it as it is for now, but it is some place for improvement.

Reply:
yes will keep in mind for further improvements.